resources/views/profile.blade.php

```diff
@@ -21,15 +21,23 @@
     </x-profile>
 
     @if ($lists->isEmpty() && $reviews->isEmpty())
-        <div class="grid flex-1 place-items-center">
-            <p class="text-slate-200">
-                @if ($isCurrentUserProfile)
-                    You don't have any content yet!
-                @else
+        @if ($isCurrentUserProfile)
+            <div class="flex flex-1 flex-col items-center justify-center gap-4">
+                <p class="text-slate-200">You don't have any content yet!</p>
```

Comment:
You could add <x-empty-state content="You don't have any content yet!"/> or i can change this when the component is merged so you dont get error msg.

Empty-state has text-slate-200

```suggestion
                <x-empty-state content="You don't have any content yet!"/>
```

Reply:
Sure. What error message?

Reply:
I was just pointing out that if you insert <x-empty-state />, it might trigger an error message since the file hasn’t been merged yet. I’d understand if you were waiting for the component, but I see you’ve made changes, which will make it easier when we refine the code later.